repo-index.yml

```diff
@@ -512,7 +512,7 @@ plugins:
   version: 1.0.0
   created: 2015-11-11
   updated: 2015-11-11
-  company: 
+  company:
```

Comment:
Resubmitted the PR with that change removed -- just my plugin addition now.

CLA submitted as well